src/rgw/rgw_zone.h

```diff
@@ -1028,18 +1028,28 @@ struct RGWPeriodConfig
 {
   RGWQuotaInfo bucket_quota;
   RGWQuotaInfo user_quota;
+  RGWQoSInfo user_qos;
+  RGWQoSInfo bucket_qos;
+  // rate limit unauthenticated user
+  RGWQoSInfo anon_qos;
 
   void encode(bufferlist& bl) const {
     ENCODE_START(1, 1, bl);
     encode(bucket_quota, bl);
     encode(user_quota, bl);
+    encode(bucket_qos, bl);
+    encode(user_qos, bl);
+    encode(anon_qos, bl);
     ENCODE_FINISH(bl);
   }
 
   void decode(bufferlist::const_iterator& bl) {
     DECODE_START(1, bl);
     decode(bucket_quota, bl);
     decode(user_quota, bl);
+    decode(bucket_qos, bl);
+    decode(user_qos, bl);
+    decode(anon_qos, bl);
```

Comment:
tthe ceph-object-corpus test is failing on `RGWPeriod` because the encode/decode of `RGWPeriodConfig` changed without bumping the version number